src/app/components/login-card/login-card.component.html

```diff
@@ -1 +1,62 @@
-<p>login-card works!</p>
+<div class="login-card-wrapper">
+
+    <mat-card class="login-card-container">
+
+        <mat-card-header class="login-card-header">
+            <mat-card-title class="login-card-title"> Yonderful Time</mat-card-title>
+        </mat-card-header>
+
+        <mat-card-content class="login-card-content">
+            <form class="login-form" [formGroup]="loginForm">
+
+                <div class="login-card-item">
+                    <mat-icon class="login-item-icon" aria-hidden="false" aria-label="Mail">mail</mat-icon>
+
+                    <mat-form-field class="login-form-item" appearance="fill">
+                        <mat-label>Email</mat-label>
+                        <input matInput type="email" placeholder="Email" formControlName="loginEmailControl">
+
+                        <mat-error *ngIf="loginForm.get('loginEmailControl')!.hasError('pattern') && 
+                                      !loginForm.get('loginEmailControl')!.hasError('required')">
+                            Please provide your <strong>firstname.lastname@tss-yonder.com</strong> mail account
+                        </mat-error>
+
+                        <mat-error *ngIf="loginForm.get('loginEmailControl')!.hasError('required')">
+                            Email is <strong>required</strong>
+                        </mat-error>
+
+                        <mat-hint>Login with your @tss-yonder.com mail account</mat-hint>
+                    </mat-form-field>
+                </div>
+
+                <div class="login-card-item">
+                    <mat-icon class="login-item-icon" aria-hidden="false" aria-label="Mail">key</mat-icon>
+
+                    <mat-form-field class="login-form-item" appearance="fill">
+                        <mat-label>Password</mat-label>
+                        <input matInput type="password" placeholder="Password" formControlName="loginPasswordControl">
+
+                        <mat-error *ngIf="loginForm.get('loginPasswordControl')!.hasError('minlength') && 
+                                    !loginForm.get('loginPasswordControl')!.hasError('required')">
+                            Password should be <strong>at least</strong> 6 characters long.
```

Comment:
this should not be displayed on a login screen. the user already has a password, he is not setting it